rfc/5/versions/1/index.md

```diff
@@ -0,0 +1,978 @@
+
+
+### "coordinateTransformations" metadata
+
+"coordinateTransformations" describe the mapping between two coordinate systems (defined by "axes").
+For example, to map an array's discrete coordinate system to its corresponding physical coordinates.
+Coordinate transforms are in the "forward" direction.
+They represent functions from *points* in the input space to *points* in the output space. 
+
+- MUST contain the field "type" (string).
+- MUST contain any other fields required by the given "type" (see table below).
+- MUST contain the field "output" (string),
+  unless part of a `sequence` or `inverseOf` (see details).
+- MUST contain the field "input" (string),
+  unless part of a `sequence` or `inverseOf` (see details).
+- MAY contain the field "name" (string).
+  Its value MUST be unique across all "name" fields for coordinate transformations.
+- Parameter values MUST be compatible with input and output space dimensionality (see details).
+
+The following transformations are supported:
+
+| Type | Fields | Description |
+|------|--------|-------------|
+| `identity` | | The identity transformation is the default transformation and is typically not explicitly defined. |
+| `mapAxis` | `"mapAxis":List[number]` | A `mapAxis` transformation specifies an axis permutation as a transpose array of integer indices that refer to the ordering of the axes in the respective coordinate system. |
+| `translation` | one of:<br>`"translation":List[number]`,<br>`"path":str` | translation vector, stored either as a list of numbers (`"translation"`) or as binary data at a location in this container (`path`). |
+| `scale` | one of:<br>`"scale":List[number]`,<br>`"path":str` | scale vector, stored either as a list of numbers (`scale`) or as binary data at a location in this container (`path`). |
+| `affine` | one of:<br>`"affine": List[List[number]]`,<br>`"path":str` | affine transformation matrix stored as a flat array stored either with json using the affine field or as binary data at a location in this container (path). If both are present, the binary values at path should be used. |
+| `rotation` | one of:<br>`"rotation":List[List[number]]`,<br>`"path":str` | rotation transformation matrix stored as an array stored either with json or as binary data at a location in this container (path). If both are present, the binary parameters at path are used. |
+| `sequence` | `"transformations":List[Transformation]` | A sequence of transformations, Applying the sequence applies the composition of all transforms in the list, in order. |
+| `displacements` | `"path":str`<br>`"interpolation":str` | Displacement field transformation located at (path). |
+| `coordinates` | `"path":str`<br>`"interpolation":str` | Coordinate field transformation located at (path). |
+| `inverseOf` | `"transform":Transform` | The inverse of a transformation. Useful if a transform is not closed-form invertible. See Forward and inverse for details and examples. |
```

Comment:
The `transform` name in the table doesn't correspond with `transformation` at https://github.com/ome/ngff/pull/350/files#diff-a3a596c95eb18d20c2f87ade777e5517c8da48b6d8968095369f5ce89345768dR396 and elsewhere.

Reply:
Fixed in b891549fa92d7a48a362880d43ffe90ab4fc1479 and [a5dba](https://github.com/jo-mueller/ngff-rfc5-coordinate-transformation-examples/commit/1fa081f208ed08077bf021240cfdbb9b774a5dba) on the examples repo